compair/api/__init__.py

```diff
@@ -58,6 +58,11 @@ def register_api_blueprints(app):
         answers_api,
         url_prefix='/api/courses/<course_uuid>/assignments/<assignment_uuid>/answers')
 
+    from .demo import demo_api
+    app.register_blueprint(
+        demo_api,
+        url_prefix='/api/demo')
+
```

Comment:
I've moved this into its own function so that it isn't mixed in with the others (needed to do this so I could register the blueprint after the fact in the tests)